core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/devfile/Devfile.java

```diff
@@ -17,14 +17,13 @@
 /** Defines Devfile. */
 public interface Devfile {
 
-  /** Returns Specification Version. It is required. */
-  String getSpecVersion();
+  /** The name of the devfile. Shortcut for {@code getMetadata().getName()}. */
+  default String getName() {
+    return getMetadata().getName();
```

Comment:
This would be safer if rewrite it in the following style:
```java
return getMetadata() == null ? null : getMetadata().getName();
```

Reply:
nice catch